llvm/lib/Target/AIE/AIEWawRegRewriter.cpp

```diff
@@ -0,0 +1,461 @@
+//===----- AEIWaWRegRewriter.cpp - Rewrite Register to not include multiple Defs
```

Comment:
I think `clang-format` broke this like because it is too long (more than 80 columns).

Reply:
fixed,